reframe/frontend/autodetect.py

```diff
@@ -48,15 +48,19 @@ def __enter__(self):
             tempfile.mkdtemp(prefix='rfm.', dir=self._prefix)
         )
         paths = ['bin/', 'reframe/', 'bootstrap.sh', 'requirements.txt']
-        for p in paths:
-            src = os.path.join(rfm.INSTALL_PREFIX, p)
-            if os.path.isdir(src):
-                dst = os.path.join(self._workdir, p)
-                osext.copytree(src, dst, dirs_exist_ok=True)
-            else:
-                shutil.copy2(src, self._workdir)
-
-        return self._workdir
+        use_pip = False
+        try:
+            for p in paths:
+                src = os.path.join(rfm.INSTALL_PREFIX, p)
+                if os.path.isdir(src):
+                    dst = os.path.join(self._workdir, p)
+                    osext.copytree(src, dst, dirs_exist_ok=True)
+                else:
+                    shutil.copy2(src, self._workdir)
+        except OSError:
```

Comment:
Maybe restrict it here:

```suggestion
        except FileNotFoundError:
```